app/javascript/components/card-table.jsx

```diff
@@ -34,14 +40,18 @@ const CardTable = props => {
     for (const [columnName, cards] of Object.entries(cardTypePairs)) {
       content.push(
         <div key={`${columnName}_column`} className={columnClass}>
-          <CardColumn key={columnName} kind={columnName} initCards={cards} />
+          <CardColumn
+            key={columnName}
+            kind={columnName}
+            initCards={cards}
+            currentUser={findCurrentUser(user, users)}
```

Comment:
I think `currentUser` should be moved to context to be available everywhere below the components tree